docs/learn/learn-proxies.md

```diff
@@ -273,5 +177,71 @@ The required deposit amount for one proxy is equal to:
 {{ polkadot: <RPC network="polkadot" path="consts.proxy.proxyDepositFactor" defaultValue={330000000} filter="humanReadable"/> :polkadot }}
 {{ kusama: <RPC network="kusama" path="consts.proxy.proxyDepositFactor" defaultValue={110000000} filter="humanReadable"/> :kusama }} * num_proxies
 
-## Resources
-[Proxy pallet documentation](https://crates.parity.io/pallet_proxy/index.html)
+## Anonymous Proxies (aka key-less anonymous accounts)
+
+:::info
+
+To create an **anonymous proxy account** check out [this support article](https://support.polkadot.network/support/solutions/articles/65000182196).
+
+:::
+
+:::note Explainer video on anonymous proxies
+
+Learn more about anonymous proxies from our
+[technical explainer video](https://www.youtube.com/watch?v=iWq53zXo7dw&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8&index=28&ab_channel=Polkadot).
+
+:::
+
+:::danger Risk of loss of funds
+
+Read carefully the text below and before performing any action using anonymous proxies on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, experiment on the Westend testnet.
+
+:::
+
+Anonymous proxies are not real proxies. Proxies that we described so far are _existing accounts_ assigned as proxies by a primary account. These proxies act on behalf of the primary account reducing the exposure of the primary account's private key. Remember, the more often we use an account's private key to sign for different things, the more we expose such key to the internet increasing the visibility of that account. The purpose a proxies is thus to draw the attention of potential attackers away from the primary account, as proxies' private keys will be used most of the time to perform actions on behalf of the primary account. We can imagine proxies as bodyguards of a VIP, loyal and ready to risk their life to ensure the VIP's protection.
```

Comment:
"are not real proxies"

I would say "they are very different from other proxy types".

also change "such key" to "that key" and add a comma after "internet".

Reply:
There are actually a few more grammatical errors here - I believe we have Grammarly access, please talk to Radha to get it installed.

Reply:
Asked in the Element chat about Grammarly. The other points are fixed now.